keyboards/royal_kludge/rk61/rgb/info.json

```
@@ -0,0 +1,332 @@
+{
+  "keyboard_name": "RK61 RGB",
+  "url": "https://www.lazada.sg/products/royal-kludge-rk61-usb-wired-bluetooth-dual-mode-60-rgb-backlit-mechanical-gaming-keyboard-i1341602754.html",
+  "maintainer": "euwbah",
```

```suggestion
  "maintainer": "euwbah",
```

Try adding yourself here too if you want to become a maintainer for this keyboard.

---

I've added myself as a keyboard maintainer.